packages/recommendations-js/README.md

```diff
@@ -60,7 +97,7 @@ The function accepts all the [shared props](#shared-props) and the following:
 
 ##### `container`
 
-> `string | HTMLElement` | **required**
+> `string | HTMLElement` |
```

Comment:
```suggestion
> `string | HTMLElement`
```

Reply:
Fixed in: https://github.com/algolia/ui-components/pull/28/commits/0aef9188e2877989fb4c919766a0265db64f24e4